core/src/curl-ev/easy.cpp

```diff
@@ -34,106 +33,200 @@
 USERVER_NAMESPACE_BEGIN
 
 namespace curl {
-namespace {
+namespace detail {
```

Comment:
There are too many changes, many of them are unrelated to the task.

Stick to the rule ”one problem - one PR". Otherwise we wont be able to review the PR in a reasonable time and we wont be able to merge it (because it wouod definitely break somerlthin in our internal code base).

Revert all the changes that are not related to the Form deprecation problem